### PR TITLE
Aligned delegates and AVCs rename

### PR DESCRIPTION
**Pull request by hernandoagf**
TODO:
- ~~`modules/app/components/ResourceBox.tsx` lines 104 and 112: change URLs to the corresponding ones for AVCs and Aligned Delegates.~~ Done.

### CODE REVIEW

File: cypress/integration/delegates/delegates.page.spec.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe('Delegates Page', () => {
     setAccount(TEST_ACCOUNTS.normal, () => {
       // Find the shadow delegates
       cy.contains('Shadow Delegates').should('be.visible');
-      cy.contains('Constitutional Delegates').should('be.visible');
+      cy.contains('aligned Delegates').should('be.visible');
```
Comment (b-pmcg) [inline cypress/integration/delegates/delegates.page.spec.ts:28]: This might be looking for capital-A "Aligned Delegates"

**Comment by b-pmcg** [comment]
looks good so far

File: cypress/integration/delegates/delegates.page.spec.ts (hunk 1 above)
Comment (hernandoagf) [reply cypress/integration/delegates/delegates.page.spec.ts:28]: Thanks!

File: modules/app/components/ResourceBox.tsx [hunk 2]
```diff
@@ -100,16 +100,16 @@ const resources: Record<ResourceType, Resource> = {
         url: 'https://manual.makerdao.com/delegation/overview/what-is-delegation'
       },
       {
-        linkTitle: "What's a Constitutional Voter Committee(CVC)?",
-        url: 'https://mips.makerdao.com/mips/details/MIP101#5-constitutional-voter-committees-cvcs-'
+        linkTitle: "What's an Aligned Voter Committee(AVC)?",
+        url: 'https://mips.makerdao.com/mips/details/MIP101#2-5-aligned-voter-committees-gov5'
       },
       {
         linkTitle: 'What is the Maker Constitution?',
```
Comment (tyler17) [inline modules/app/components/ResourceBox.tsx:107]: should this have a different title? The MIP101 title is Maker Atlas Immutable Alignment Artifact

**Review by tyler17 — DISMISSED** [review]
LMGT, just had one question

File: modules/app/components/ResourceBox.tsx (hunk 2 above)
Comment (hernandoagf) [reply modules/app/components/ResourceBox.tsx:107]: Good catch!

**Review by b-pmcg — APPROVED** [review]

**Pull request merged**